courseify/courses/templates/courses/base.html

```diff
@@ -1,11 +1,28 @@
 <!-- {% load static %} -->
 <!DOCTYPE html>
 <html lang="en">
+<<<<<<< HEAD
```

Comment:
@AdityaAshvin These are due to conflicts.

Reply:
okay. I'll fix them.